### ISSUE TITLE
score functions - sklearn compatibility - parameter order

### ISSUE THREAD

**Issue by FabianWalocha** [sh0rtcircuit/python-probabilistic-regression-tools#2]
any metric in sklearn.metrics (e.g. f1_score) takes order of parameters as (y_true, y_pred), while all scores here use (y_pred, y_true)
